features/command_line_interface/options.feature

```diff
@@ -60,7 +60,7 @@ Feature: Reek can be controlled using command-line options
       Text format options:
               --[no-]color                 Use colors for the output (default: true)
           -V, --[no-]empty-headings        Show headings for smell-free source files (default: false)
-          -U, --[no-]wiki-links            Show link to related wiki page for each smell (default: true)
+          -U, --[no-]documentation            Show link to related wiki page for each smell (default: true)
```

Comment:
-> 'related documentation page'

Reply:
Good point, I also noticed that i missed quite some other internal references to "wiki", lemme fix this.

Reply:
Oh boy, there are a LOT of references to wiki that we had missed 😆